packages/ott-vis-panel/src/components/CorePanel.tsx

```diff
@@ -42,6 +51,22 @@ export const CorePanel: React.FC<Props> = props => {
 	return <CoreData {...props} />;
 };
 
+const Legend: React.FC = () => {
+	//const colors = useColorProvider();
+	// implement hook into legend listing
+	return (
+		<div className={useStyles2(getStyles).legendBox}>
```

Comment:
don't use emotion because its annoying, create a `legend.css` and import it with `import "./legend.css";`